Patch pulled from upstream repo, author Samuel Thibault.
Goes away with 0.8.7 or whatever their next version is.
diff --git src/api/python/speechd_config/config.py.in src/api/python/speechd_config/config.py.in
index a59ef79a..e9b2ae53 100644
--- src/api/python/speechd_config/config.py.in
+++ src/api/python/speechd_config/config.py.in
@@ -156,6 +156,7 @@ def setup_argparse():
     parser.add_argument('-D', '--debug', dest='debug',
             action="store_true", default=False,
             help=_("Debug a problem and generate a report"))
+    return parser
 
 class Tests:
     """Tests of functionality of Speech Dispatcher and its dependencies
@@ -750,7 +751,7 @@ you have to start it manually to continue."""))
 
 def main():
     options = setup_argparse()
-    options.parse()
+    options = options.parse_args()
     test = Tests()
     configure = Configure(test) # that should possibly be refactored, test should not be passed
 
